backend/src/Squidex.Web/Resources.cs

```diff
@@ -22,6 +22,8 @@ public sealed class Resources
         // Contents
         public bool CanReadContent(string schema) => IsAllowedForSchema(P.AppContentsRead, schema);
```

Comment:
We have to change everything to "own", please do not remove the method names to keep the SRP.